plugins/org.eclipse.emf.mwe.core/src/org/eclipse/emf/mwe/internal/core/debug/processing/handlers/VariablesRuntimeHandler.java

```diff
@@ -232,7 +232,7 @@ private void cleanFramesCache() {
 	/**
 	 * push the syntax element onto the stack
 	 * 
-	 * @see org.eclipse.emf.mwe.internal.core.debug.processing.EventHandler#preTask(java.lang.Object, int)
+	 * @see org.eclipse.emf.mwe.core.debug.processing.EventHandler#preTask(java.lang.Object, java.lang.Object, int)
```

Comment:
remove java.lang?

Reply:
i did not change the existing pattern. i just fixed places inline to be consistent
=> please open separate pr